### DISCUSSION TITLE
How can I get the free space on the SD card

### DISCUSSION THREAD

**Discussion by Semsem-220109** [stm32duino/Arduino_Core_STM32#2183] [Q&A]
Hi, I want to get the free space available on the SD card, but I couldnt figure out how to do it. Can someone help me ?

**Comment by fpistm** [comment]
Hi @Semsem-220109 
There is no dedicated function for this.
Anyway, you can do this on your own, I guess:
Get SD size:
https://github.com/stm32duino/STM32SD/blob/8e85a6d2011699b660b76e38826de6c139a22c51/examples/CardInfo/CardInfo.ino#L64C7-L79
Then sum all files size:
https://github.com/stm32duino/STM32SD/blob/main/examples/listfiles/listfiles.ino#L78C21-L78C21

**Reply by Semsem-220109** [reply]
Ok thanks, and is it planned to be implemented in the future ? Just asking !

**Reply by fpistm** [reply]
No. This answer is application level.
